pkg/ansible/watches/watches_test.go

```diff
@@ -599,3 +601,98 @@ func TestAnsibleVerbosity(t *testing.T) {
 		})
 	}
 }
+
+// Test the func getFullRolePath when no envVar is set.
+func Test_getFullRolePath(t *testing.T) {
+	// Mock default Full Path based in the current directory
+	rolesPath := filepath.Join(projutil.MustGetwd(), "roles")
+
+	type args struct {
+		path string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Should work with default values",
+			args: args{
+				path: "Foo",
+			},
+			want: filepath.Join(rolesPath, "Foo"),
+		},
+		{
+			name: "Should work with full path",
+			args: args{
+				path: filepath.Join(rolesPath, "Foo"),
+			},
+			want: filepath.Join(rolesPath, "Foo"),
+		},
+		{
+			name: "Should work with relative path",
+			args: args{
+				path: "relative/Foo",
+			},
+			want: filepath.Join(rolesPath, "relative/Foo"),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := getFullRolePath(tt.args.path); got != tt.want {
+				t.Errorf("Error to check getFullRolePath() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+// Test the func getFullRolePath when the custom flag and/or env var was set.
+func Test_getFullRolePathWithEnvVarSet(t *testing.T) {
+	// Mock customized Path
+	customPath := "customized/myroles"
+	os.Setenv("ANSIBLE_ROLES_PATH", customPath)
```

Comment:
Can we add a couple of other tests with the following paths:
- "nonexistent/roles"
- "customized/myroles:nonexistent/roles"
- "nonexistent/roles:customized/myroles" 
- "nonexistent1/roles:nonexistent2/roles"

Reply:
The tests were improved. 
We are testing the func getFullRolePath with and whiteout the envVar + we are checking the load well with the successful scenarios.